### PR TITLE
Update wording in ES upgrade changelog, add EOL

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
Followup to https://github.com/dimagi/commcare-cloud/pull/3580

Diff shows better if you look at the individual commits (🐡)

### CODE REVIEW

**Comment by dannyroberts** [comment]
@esoergel I think once this is merged, you're good to kick off the announcement etc like we talked bout in https://github.com/dimagi/commcare-hq/pull/25898.

Good with April 2?

**Comment by dannyroberts** [comment]
@sanjay2916 FYI, a few changes to hopefully make things more accessible to third parties

File: changelog/0032-upgrade-to-elasticsearch-2.4.6.yml [hunk 1]
```diff
@@ -1,102 +1,115 @@
...
+      "cluster.routing.allocation.enable": "none"
     }
   }
   '
   ```
 
   5. Perform a synced flush
   ```bash
-  curl -X POST "<ES-IP>:9200/_flush/synced?pretty"
+  curl -X POST "$(cchq ${ENV} lookup elasticsearch:0):9200/_flush/synced?pretty"  # safe to reissue a few times if it fails
```
Comment (esoergel) [inline changelog/0032-upgrade-to-elasticsearch-2.4.6.yml]: > safe to reissue a few times if it fails

NIce touch - failure mode help is really valuable

File: changelog/0032-upgrade-to-elasticsearch-2.4.6.yml [hunk 2]
```diff
@@ -1,102 +1,115 @@
...
   ```bash
-  commcare-cloud <env> run-shell-command all "service monit stop" -b --limit=elasticsearch
-  commcare-cloud <env> run-shell-command all "service elasticsearch stop" -b --limit=elasticsearch
+  commcare-cloud ${ENV} run-shell-command all "service monit stop" -b --limit=elasticsearch
+  commcare-cloud ${ENV} run-shell-command all "service elasticsearch stop" -b --limit=elasticsearch
   ```
-    
-  9. Rename the newly created 2.4.6 data directory with diffrent name and then rename the 1.7.6 data folder with new version 2.4.6
+
+  9. Rename the newly created 2.4.6 data directory so it's out of the way, and make a backup of the 1.7.6 data directory, and then move it to the new 2.4.6 data directory location
```
Comment (esoergel) [inline changelog/0032-upgrade-to-elasticsearch-2.4.6.yml]: If we're recommending a backup, shouldn't it be done earlier?  Like right after bringing down the site?

File: changelog/0032-upgrade-to-elasticsearch-2.4.6.yml [hunk 3]
```diff
@@ -1,102 +1,115 @@
...
 
 update_steps: |
 
+  Before you start, make sure that your disk usage is well below 50%,
+  or that you have a place to back up to on another disk or remotely.
+
+  0. For commands including `${ENV}` below to work, make sure to set `ENV` to the name of your environment
+  ```
+  ENV=${ENV}
+  ```
```
Comment (esoergel) [inline changelog/0032-upgrade-to-elasticsearch-2.4.6.yml]: Could also add to this something like
```
ES_HOST=$(cchq ${ENV} lookup elasticsearch:0)
```
That'd make the below commands a bit easier to follow if something goes awry

Comment (dannyroberts) [reply changelog/0032-upgrade-to-elasticsearch-2.4.6.yml]: Good idea, ✔️

File: changelog/0032-upgrade-to-elasticsearch-2.4.6.yml (hunk 2 above)
Comment (dannyroberts) [reply changelog/0032-upgrade-to-elasticsearch-2.4.6.yml]: Also good idea, ✔️

**Pull request merged**